scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -435,7 +431,8 @@ def create_branch(
     common.run_cmd(GIT_CMD_CHECKOUT.split(' '))
     common.open_new_tab_in_browser_if_possible(
         'https://github.com/oppia/oppia/compare/develop...%s:%s?'
-        'expand=1' % (github_username, target_branch))
+        'expand=1&title=Update authors and changelog for v%s' % (
```

Comment:
Just to check, did you manually test this URL? (You can try it with an existing branch.)

Wanted to make sure the encoding is correct.

Reply:
Yes, tested. It works fine.